dev-libs/efl/Manifest

```diff
@@ -1,2 +1,3 @@
 DIST efl-1.17.0.tar.xz 43485360 BLAKE2B 97a79bda40658eac6e50886a01821bb8c98d6f56d18fcd51dc3827154506064bec3bf665c8f471919aedb03ae83ce4045293afa9a0bf8d3eaf947221ef7cba9e SHA512 e4f28f938daf4ec1cb3d761407a443e401b59c6bb0f9fa6cd771dbd256b95f4a32a5f9e6bd8ad8328708a382f732e67abd1a23c7be2ff222f08bc8135a2435da
```

Comment:
Please shorten the commit message to only roughly describe changes. There is really no need or reason to make it into a detailed changelog, and it is going to occupy a lot of screen space for anyone looking at the log.

Reply:
Done